lib/python/qmk/cli/docs.py

```diff
@@ -0,0 +1,20 @@
+"""Serve QMK documentation locally
+"""
+import http.server
+from os import chdir
+
+from milc import cli
+
+
+@cli.argument('-p', '--port', default=8080, help='Port number to use.')
+@cli.subcommand('Local instance of QMK docs')
+def docs(cli):
+    """Spin up a local HTTPServer instance for the QMK docs.
+    """
+    cli.log.info("Serving QMK docs locally on port %d.", cli.config.docs.port)
+    cli.log.info("Press Control+C to exit.")
+
+    chdir('docs')
+
+    httpd = http.server.HTTPServer(('', cli.config.docs.port), http.server.SimpleHTTPRequestHandler)
```

Comment:
Not a blocker, but you _could_ pass `directory` to `SimpleHTTPRequestHandler` instead of using the `chdir`.

Reply:
Ah, I didn't see anything like that but it turned out I was looking at `BaseHTTPRequestHandler` instead. Thanks!

Reply:
It seems like I have to subclass the handler in order to do this, because I'm not actually instantiating it here. Or is there a better way?